C++/include/algorithm/dynamic_programming/longest_increasing_subsequence.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+      Longest Increasing Subsequence
+      ------------------------------
+      Given an unsorted array of integers, return the longest increasing subsequence length
+
+      Time complexity
+      --------------
+      O(N^2), where N is size of array
+
+      Space complexity
+      ---------------
+      O(N), where N is size of array
+
+      Author
+      ------
+      Shikhar Jaiswal(@shikherjaiswal)
+*/
+
+#ifndef longest_increasing_subsequence_hpp
+#define longest_increasing_subsequence_hpp
+
+#include <vector>
+
+/*
+    ==========================================================================
+    Public interface
+    ==========================================================================
+*/
+
+
+/*
+    lis
+    ----------------
+    Returns the length of longest increasing subsequence of vector -arr
+*/
+
+int lis(std :: vector< int > arr, int n) {
```

Comment:
* Rename the `lis` function to `longest_increasing_subsequence`.
* Remove unnecessary whitespace around `::`, `<`, `>` (please also make this change in other files you've added).
* What is `n` for? If it's value is always supposed to be the length of `arr` then it is not required. Otherwise, rename it to something more meaningful (like `max_index` or something).